# Antoine Limasset

**Chargé de recherche CNRS**  
I am a **research associate in [BONSAI](https://www.cristal.univ-lille.fr/bonsai/) team (Lille, France)**. My work focuses on **methods and data structures** in **sequence bioinformatics**

## Education



- **2014:** M.Sc. in Computer Science, École Normale Supérieure de Rennes

- **2012:** B.Sc. in Computer Science, École Normale Supérieure de Cachan

- **2008-2011:** MPSI-MP*-MP*, Lycée Roosvelt et Lycée Clemenceau, Reims

- **2008:** Baccalauréat Scientifique, Saint Dizier

## Experience

- **2018  current:** Researcher at CNRS, CRIStAL, Lille.  
  Role: Chargé de recherche au sein de l'equipe Bonsai

- **2017 2018:** PostDoctoral researcher at Université Libre de Bruxelles, Belgique.  
  *Project:* de novo assembly of heterozygous genomes

  - **2014 2017:** Ph.D in Computer Science, Université de Rennes 1. Supervised by Pierre Peterlongo and Dominique Lavenier.  
  *Title:* Novel approaches for the exploitation of high throughput sequencing data

## Publications

- **2023:** Locality-Preserving Minimal Perfect Hashing of k-mers, ISMB
- **2023:** Scalable sequence database search using Partitioned Aggregated Bloom Comb-Trees, ISMB/Recomb-Seq
- **2022:** Toward optimal fingerprint indexing for large scale genomics, WABI
- **2022:** Critical Assessment of Metagenome Interpretation: the second round of challenges, Nature Methods
- **2021:** STRONG: metagenomics strain resolution on assembly graphs, Genome Biology
- **2021:** BLight: Efficient exact associative structure for k-mers, Recomb-seq/Bioinformatics
- **2021:** Scalable long read self-correction and assembly polishing with multiple sequence alignment, Recomb-seq/Scientific report
- **2020:** Toward perfect reads: self-correction of short reads via mapping on de Bruijn graphs, Recomb-seq/Bioinformatics
- **2020:** ELECTOR: evaluator for long reads correction methods, NAR Genomics and Bioinformatics
- **2017:** Fast and scalable minimal perfect hashing for massive key sets, Symposium on Experimental Algorithms
- **2016:** Compacting de Bruijn graphs from sequencing data quickly and in low memory, ISMB/Bioinformatics
- **2016:** Read mapping on de Bruijn graphs, BMC Bioinformatics
- **2016:** A resource-frugal probabilistic dictionary and applications in bioinformatics, Discrete Applied Mathematics
- **2014:** On the representation of De Bruijn graph, RECOMB

## Supervising

- **2019 - current:** Ph.D student, Coralie Rohmer.  
  *Project:* Working on multiple alignment algorithms adapted to third generation sequencing
- **2022 - current:** Ph.D student, Léa Vandamme.  
  *Project:* Working on Third generation sequencing datasets indexing

## Conference committee

- **2021:** Organizational committee, SPIRE
- **2020-2023:** Program committee, RECOMB
- **2020-2023:** Program committee, ACM-BCB
- **2020-2023:** Program committee, ECCB/ISMB

## Grant

- **2021:** ANR JCJC (young researchers).  
  *Project:* Adequate graph structures for third-generation sequencing data exploration
